.travis.yml

```diff
@@ -42,6 +40,41 @@ matrix:
     # eth-tester-pyevm
     - python: "3.5"
       env: TOX_POSARGS="-e py35-integration-ethtester" ETHEREUM_TESTER_CHAIN_BACKEND=eth_tester.backends.PyEVMBackend
+    # Python 3.6 testing
+    # go-ethereum
+    - python: "3.6"
+      env: TOX_POSARGS="-e py36-integration-goethereum" GETH_VERSION=v1.6.0
+    - python: "3.6"
+      env: TOX_POSARGS="-e py36-integration-goethereum" GETH_VERSION=v1.6.1
+    # 1.6.2 and 1.6.3 are failing due to go-ethereum internal errors.
+    - python: "3.6"
+      env: TOX_POSARGS="-e py36-integration-goethereum" GETH_VERSION=v1.6.4
+    - python: "3.6"
+      env: TOX_POSARGS="-e py36-integration-goethereum" GETH_VERSION=v1.6.5
+    - python: "3.6"
+      env: TOX_POSARGS="-e py36-integration-goethereum" GETH_VERSION=v1.6.6
```

Comment:
I believe @carver was looking to drop testing against everything but `v1.7.2` since it's the *only* version of go-ethereum that is currently viable to be run on the mainnet.

Reply:
Correct. #428 will push that change for the 3.5 python environments, so you can leave them out of the 3.6 ones.